symphony/content/content.blueprintsdatasources.php

```diff
@@ -331,7 +331,7 @@ public function __form()
         $fieldset = new XMLElement('fieldset');
         $this->setContext($fieldset, array('sections', 'system'));
         $fieldset->appendChild(new XMLElement('legend', __('Error Conditions')));
-        $p = new XMLElement('p', __('Enabling these conditions will send a <code>404 page not found</code> error.'));
+        $p = new XMLElement('p', __('If enabled conditions are met, a <code>"404 page not found"</code> error will be sent.'));
         $p->setAttribute('class', 'help');
```

Comment:
Ah! I see your point!

> I would like to hear what native speakers think about it.

@brendo seemed fine with return, then. I'll change it.

Thanks for playing with me @michael-e